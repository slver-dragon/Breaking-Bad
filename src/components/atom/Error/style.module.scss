.error {
  width: 60%;
  padding: 5rem 2rem;
  margin: 10rem auto;
  display: flex;
  justify-content: center;
  align-items: center;
  font-size: 1.3rem;
  font-weight: bold;
  background-color: #fefefe;
  border-radius: 0.5rem;
}