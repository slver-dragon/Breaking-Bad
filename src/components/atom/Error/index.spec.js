import React from "react";
import { shallow } from "enzyme";
import Error from "../Error";

describe("Error atom testing:", () => {
  test("Is render", () => {
    const component = shallow(<Error textError="Test Error" />);
    expect(component).toMatchSnapshot();
  });
});
