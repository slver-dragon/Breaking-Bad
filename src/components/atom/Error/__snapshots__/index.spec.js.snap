// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error atom testing: Is render 1`] = `
<div
  className="error"
>
  Test Error
</div>
`;
